CMP1903_A1_2324/Game.cs

```diff
@@ -8,14 +8,25 @@ namespace CMP1903_A1_2324
 {
     internal class Game
     {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
+        private static Testing test = new Testing(); //A new object for Testing is created here, so that tests can be made as soon as possible
+        public void DieGame() //DieGame is the starting point of the program, as seen from Program.cs, which means that these are the first pieces of code to run
+        {
+            Die dieOne = new Die(); //creates a die object, which will be used to roll a number between 1 and 6
+            dieOne.Roll(); //uses the die object to start a unique roll method for it
+            int DieOneValue = dieOne.DieValue; //returned integer is set as a value for the die object
 
-        //Methods
+            Die dieTwo = new Die(); //creates a new die object
+            dieTwo.Roll(); //uses the die object to start a unique roll method for it
+            int DieTwoValue = dieTwo.DieValue; //returned integer is set as a value for this unique die object
 
+            Die dieThree = new Die(); //creates a new die object
+            dieThree.Roll(); //uses the die object to start a unique roll method for it
+            int DieThreeValue = dieThree.DieValue; //returned integer is set as a value for this unique die object
```

Comment:
I like your way of getting 3 dice objects, just simply copy pasting the code over, instead of over complicating it,
As mentioned by the other reviewer, you could have used an array to make these objects but that would require more testing and be more harder to do.

But it does work in the end which is what matters.